payments/admin.py

```diff
@@ -190,10 +190,7 @@ def get_queryset(self, request):
         return super().get_queryset(request).filter(state=models.Order.STATE.REFUNDED)
 
 
-@admin.register(models.Discount)
```

Comment:
This line (193) needs to go back - the DiscountAdmin is unregistered now so can't be used.